mmda/types/document.py

```diff
@@ -263,7 +263,7 @@ def from_json(cls, doc_dict: Dict) -> "Document":
         # 2) convert span group dicts to span gropus
         field_name_to_span_groups = {}
         for field_name, span_group_dicts in doc_dict.items():
-            if field_name not in doc.REQUIRED_FIELDS:
+            if field_name not in doc.SPECIAL_FIELDS:
```

Comment:
Removing `Images` from `REQUIRED_FIELDS` caused an error here, so I restored it, renaming to `SPECIAL` since images are not actually required. I think the check in `annotate()` above is correct, since we don't want users to call that method to annotate the images.